2018/002-update-channels.md

```diff
@@ -0,0 +1,184 @@
+ Title
+
+| Field  | Value      |
+|:-------|:-----------|
+| Status | Draft      |
+| Date   | 2018-07-31 |
+
+## Introduction
+
+### Problem description
+
+Updates can be disruptive for a customer's application stack (ex. Kubernetes
+upgrade from 1.10.x to 1.11.x, default pod security policy introduction).
+
+Right now there is no visibility of what is inside an update and furthermore
+there is no possible way around intrusive updates by having the ability to only
+apply critical bug fixes and security updates. An admin needs to apply all updates
+that come through a single channel.
+
+In order to separate intrusive updates in SUSE CaaS Platform, we need different
+update channels. This allows the separation of possibly disruptive updates
+into their own channels, and lets the admin continue applying bugfixes and security
+updates without the need to apply intrusive, new features. The admin can also see
+what is inside an update to be able to decide when to apply it.
+
+The channels will also act as a barrier to avoid that a user runs k8s 1.8 and
+upgrades right away to 1.11 in one step.
+
+### Concrete Example
+
+CaaS Platform installations will still have two repositories active. The Update and
+the Pool Channel. Both have the same version. This repositories, hereinafter called "channel"
+are associated with a certain version of the product, for example: `3.1`.
+The channel is going to provide regular maintenance updates (eg: security fixes) and bug
```

Comment:
The update channel is going to ...
(else you don't know if the update or pool channel is meant)